src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double armLength;
+    private double bucketCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. It's good practice to make fields private and provide getters and setters for accessing them.